### PR TITLE
Improving readability of Maxmind test classes

### PR DESCRIPTION
**Pull request by SavvasMichael**
Improves the readability of the Maxmind unit tests. No changes to behavior included.

### CODE REVIEW

File: src/test/java/technology/dice/dicewhere/parsing/provider/maxmind/MaxmindLineParserTest.java [hunk 1]
```diff
@@ -159,45 +170,45 @@ public void ipV6LineWithoutOriginal() throws LineParsingException {
...
-      throw e;
+      assertEquals(line, e.getOffendingLine().getLine());
+      assertEquals(1, e.getOffendingLine().getLineNumber());
     }
   }
 
-  @Test(expected = LineParsingException.class)
-  public void wrongCIDRFormat() throws LineParsingException {
-    MaxmindLineParser maxmindParser = new MaxmindLineParser(locationNames);
+  @Test()
```
Comment (stykiaz) [inline src/test/java/technology/dice/dicewhere/parsing/provider/maxmind/MaxmindLineParserTest.java]: we can loose the brackets

File: src/test/java/technology/dice/dicewhere/parsing/provider/maxmind/MaxmindLineParserTest.java [hunk 2]
```diff
@@ -159,45 +170,45 @@ public void ipV6LineWithoutOriginal() throws LineParsingException {
...
-      throw e;
+      assertEquals(line, e.getOffendingLine().getLine());
+      assertEquals(1, e.getOffendingLine().getLineNumber());
     }
   }
 
-  @Test(expected = LineParsingException.class)
-  public void impossibleCIDR() throws LineParsingException {
-    MaxmindLineParser maxmindParser = new MaxmindLineParser(locationNames);
+  @Test()
```
Comment (stykiaz) [inline src/test/java/technology/dice/dicewhere/parsing/provider/maxmind/MaxmindLineParserTest.java]: same here

File: src/test/java/technology/dice/dicewhere/parsing/provider/maxmind/MaxmindLineParserTest.java (hunk 1 above)
Comment (SavvasMichael) [reply src/test/java/technology/dice/dicewhere/parsing/provider/maxmind/MaxmindLineParserTest.java]: Fixed

File: src/test/java/technology/dice/dicewhere/parsing/provider/maxmind/MaxmindLineParserTest.java (hunk 2 above)
Comment (SavvasMichael) [reply src/test/java/technology/dice/dicewhere/parsing/provider/maxmind/MaxmindLineParserTest.java]: Fixed

**Pull request merged**